docstub.toml

```
@@ -19,3 +19,5 @@ DataTree = "xarray.DataTree"
 # TODO: define a meta/pseudo type for DataTree-like
 DataTree-like = "xarray.DataTree"
 DataArray = "xarray.DataArray"
+scalar = "int"
```

```suggestion
scalar = "numbers.Number"
```

Plus above in known imports add `numbers = {import = "numbers"}`. Full context, `numbers` is part of the standard library, and numpy adds its dtypes as aliases or subclasses of standard library elements when possible, so this will cover int, float but also float32 and numpy specific types. Ref https://numpy.org/doc/stable/reference/generated/numpy.isscalar.html

---

I saw it in your previous comment, but it didn't work. I added `numebrs` to `known_imports` and it did. :)